glossary.md

```diff
@@ -0,0 +1,18 @@
+# Vidarr Glossary
+
+- analysis output - the data generated by a workflow run. Vidarr understands two kinds of output: files and URLs
+- external identifier - a reference to external (LIMS) data. It has a _provider_, the name of the data store that is the source for an external identifier, and an _id_ which is the store-specific name for this data. The same ID can be re-used across multiple providers.
+- external key - a reference to a specific version of external (LIMS) data. This has the same data as an _external identifier_ plus a dictionary of versions. Each version has a _version key_ and a _version value_.
```

Comment:
Many versions. I'm not sure what the best way to explain this is. So, Pinery will produce something like:
```
{
  "provider": "pinery-miso",
  "id": "1234_1_LDI1234",
  "versions": { "pinery-hash-7": "012456789abcdef" }
}
```

Now, Shesmu has the ability to generate a signature based on the data the olive read, so by the time it makes it to Vidarr, it will look something like:

```
{
  "provider": "pinery-miso",
  "id": "1234_1_LDI1234",
  "versions": { "pinery-hash-7": "012456789abcdef", "shesmu-sha1": "cafe123" }
}
```
And when we hit the next Pinery format, we can push an update so it will have both `"pinery-hash-7"` and `"pinery-hash-8"`.

This is a bit much for a glossary. Maybe another doc? It's already partially described in the architecture doc.

Reply:
If it's described in the architecture doc then a link there from this definition would probably be good. Otherwise I think just dropping the 'a reference to a specific version' is fine, since this all seems to make it sound like that's false anyway?